aap-common/assembly-appendix-release-notes.adoc

```diff
@@ -0,0 +1,62 @@
+endif::product_GCP[]
+
+[NOTE]
+====
+Automation Mesh and Event Driven Automation are not yet supported by the self-managed Ansible on Cloud offerings at this time.
+====
+
+[discrete]
+== Enhancements
+
+ifdef::product_AWS[]
+The release version2.4.20230630 of {AAPonAWS} includes the following enhancements:
+endif::product_AWS[]
+ifdef::product_GCP[]
+The release version2.4.20230630 of {AAPonGCP} includes the following enhancements:
+endif::product_GCP[]
+
+* Support for {PlatformNameShort} 2.4 has been added.
+* Added internal encryption to the web servers
+** Addition of custom certificates is now supported.
+* Support has been added for custom tagging and labelling.
+ifdef::product_AWS[]
+** Support has been added to AWS to add or remove tag support for resources owned by the deployment.
+endif::product_AWS[]
+ifdef::product_GCP[]
+** Support has been added to GCP to add or remove tag support for resources owned by the deployment.
+* {AAPonGCP} now has operational playbooks to add and remove extension nodes.
+endif::product_GCP[]
+* There are improvements to backup and restore functionality.
+** Support has been added for taking multiple backups.
+** An operational playbook has been added to list available backups.
+** An operational playbook has been added to delete a selected backup.
+** The capability to restore an existing VPC has been added.
+* Operational functionality to check the current version has been added.
+** Operational playbooks now check to ensure that the {PlatformNameShort} environment they are operating upon is at the same version before beginning their operation.
+
+
+[discrete]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in {PlatformNameShort} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+The following is a list of major functionality deprecated and removed within {PlatformNameShort} 2.3:
+
+* On-premise component Automation Services Catalog is now removed from {PlatformNameShort} 2.4 onwards.
+* With the {PlatformNameShort} 2.4 release, the Execution Environment container image for Ansible 2.9 (ee-29-rhel-8) is no longer loaded into the Automation Controller configuration by default.
+
+[discrete]
+== Additional Release Notes related to Ansible Automation Platform
+
+* Check out latest features in link:https://access.redhat.com/login?redirectTo=https%3A%2F%2Faccess.redhat.com%2Fdocumentation%2Fen-us%2Fred_hat_enterprise_linux%2F9[Red Hat Enterprise Linux 9]
+* Read more about latest link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_release_notes/index?extIdCarryOver=true&intcmp=7013a0000026H45AAE&sc_cid=7013a000003SeN0AAK[Ansible Automation Platform features]
```

Comment:
For more information about the latest {PlatformnameShort} features, read link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_release_notes/index?extIdCarryOver=true&intcmp=7013a0000026H45AAE&sc_cid=7013a000003SeN0AAK[Ansible Automation Platform features]